scripts/verify-sign.ps1

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Microsoft. All rights reserved.
+# Build script for Test Platform.
+
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory=$false)]
+    [ValidateSet("Debug", "Release")]
+    [Alias("c")]
+    [System.String] $Configuration = "Debug"
+)
+
+#
+# Variables
+#
+$rootDirectory = (Get-Item (Split-Path $MyInvocation.MyCommand.Path)).Parent.FullName
+
+#
+# Signing configuration
+#
+# Authenticode signature details
+Write-Verbose "Setup build configuration."
+$TPB_Configuration = $Configuration
+
+function Verify-NugetPackages
+{
+    Write-Log "Verify-NugetPackages: Start"
+
+    $toolsDirectory = Join-Path $rootDirectory "tools" 
+
+    # Move acquiring nuget.exe to external dependencies once Nuget.Commandline for 4.6.1 is available.
+    $nugetInstallDir = Join-Path $toolsDirectory "nuget"
```

Comment:
No need to download nuget.exe again. Our build script already does it. Consider using [this ](https://github.com/Microsoft/testfx/blob/master/scripts/common.lib.ps1#L61)function.